app/routes/project/settings/donations.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+
+const {
+  inject: { service },
+  Route
+} = Ember;
+
+export default Route.extend({
+  ajax: service(),
+
+  model() {
+    let project = this.modelFor('project');
+    return project;
+  },
+
+  afterModel(model) {
+    this.store.queryRecord('stripe-auth', {
```

Comment:
I think perhaps `this.store` is preferred but it's hard to tell what the convention is.